mars/lib/filesystem/_glob.py

```diff
@@ -153,10 +156,9 @@ def _glob2(self, dirname, pattern, dironly):  # pragma: no cover
     # If dironly is true, yields only directory names.
     def _iterdir(self, dirname, dironly):
         if not dirname:  # pragma: no cover
-            if isinstance(dirname, bytes):
-                dirname = bytes(os.curdir, "ASCII")
-            else:
-                dirname = os.curdir
+            dirname = ""
```

Comment:
For line 159, since `os.curdir` is `.`, which is not recognized by `fsspec`,  it is needed to replace `os.curdir` with an empty string.

For lines 160-161, `_iterdir` should return an empty iter if the input path refers to a file.